### ISSUE TITLE
Fullscreen option for web browser version.

### ISSUE THREAD

**Issue by BeezBumba** [jpd002/Play-#1412]
## Quick summary
Please briefly describe what is not working correctly
I apologize for saying this is an issue even thought it's a suggestion.
## System Details
Describe the ambient being used to run Play!
Information needed:
- Operating system (including the version)
- Hardware (GPU and CPU, for android you can search for your device model here https://www.gsmarena.com/ for specifications)
- Version of Play!

## Issue Details
Describe in details the problem you had and the expected behavior
None
## Screenshots/Videos
Drag and drop (or link to) videos or images that could help us understand the problem.
None

**Comment by hardBSDk** [comment]
@BeezBumba Does the full-screen mode of the web browser is enough?

**Comment by BeezBumba** [comment]
@hardBSDk Yes, I want to be able to full screen on that window that shows the game.
